src/main/java/com/easysplit/ess/user/infrastructure/persistence/validators/UserDatabaseValidator.java

```diff
@@ -59,7 +59,7 @@ private void validateUserEmailUniqueness(String email) {
         if (userEntity != null) {
             infrastructureHelper.throwIllegalArgumentException(
                     ErrorKeys.CREATE_USER_ILLEGALARGUMENT_TITLE,
-                    ErrorKeys.CREATE_USER_NOT_UNIQUE_USERNAME_MESSAGE,
+                    ErrorKeys.CREATE_USER_NOT_UNIQUE_EMAIL_MESSAGE,
                     new Object[]{ email }
```

Comment:
In both validate uniqueness for phone and e-mail you used black spaces and beginning and end = { email }, but in all other exceptions we have been using no black spaces = {email}

Reply:
I just made the change to keep cosistency